Project.toml

```diff
@@ -15,6 +15,7 @@ JSONSchema = "7d188eb4-7ad8-530c-ae41-71a32a6d4692"
 MacroTools = "1914dd2f-81c6-5fcd-8719-6d5c9610ff09"
 NodeJS = "2bd173c7-0d6d-553b-b6af-13a54713934c"
 Pkg = "44cfe95a-1eb2-52ea-b672-e2afdf69b78f"
+Query = "1a8c2f83-1ff3-5112-b086-8aa67b057ba1"
```

Comment:
I think this can/should go into `docs/Project.toml`, right?

Reply:
It is already there.
I think it has been added automatically to the root Project.toml somehow and I ignored it.